### ISSUE TITLE
Tabs.tsx `goToTab()` not working properly

### ISSUE THREAD

**Issue by Kirbo** [ant-design/ant-design-mobile-rn#1253]
https://github.com/ant-design/ant-design-mobile-rn/blob/master/components/tabs/Tabs.tsx#L259

This...
```typescript
  goToTab(index: number) {
    if (this.carousel) {
      this.carousel.goTo(index)
    }
    this.setState({ currentTab: index }, () => {
      this.state.scrollX.setValue(index * this.state.containerWidth)
    })
  }
```
...compiles to:
```typescript
{
  key: "goToTab",
  value: function goToTab(index) {
    if(this.carousel){
      this.carousel.goTo(index);
    }
    this.state.scrollX.setValue(index*this.state.containerWidth);
  }
}
```
This can be found in file `node_modules/@ant-design/react-native/lib/tabs/Tabs.js`, search for `key:"goToTab"`

And as you can see, the `setState()` is missing from it.

Could this work better?
```typescript
this.setState((state, props) => ({
  currentTab: index,
  scrollX: state.scrollX.setValue(index * state.containerWidth)
}));
```
..I don't know. Anyway, the transpiled version of the current implementation does not work (it does not transpile correctly).

**Comment by 1uokun** [comment]
The compilation result is correct, I fixed it here https://github.com/ant-design/ant-design-mobile-rn/commit/c2c59056b2d247b1c849808a95e891b2c51ee47a, just haven't updated the version

**Comment by 1uokun** [comment]
fixed in `5.0.2`
